boot/boot_test.go

```diff
@@ -3170,6 +3170,8 @@ func (s *bootConfigSuite) TestBootConfigUpdateWithGadgetAndReseal(c *C) {
 	}
 	c.Assert(m.WriteTo(""), IsNil)
 
+	s.bootloader.Updated = true
```

Comment:
was this test wrong before or is it really a change related to the PR?

Reply:
Yes, it's an artifact of the merges/rebases I did on the development branch. Thanks for spotting.